reference/5.1/Microsoft.PowerShell.Core/About/about_Remote_Variables.md

```diff
@@ -105,14 +105,37 @@ $Splat = @{ Name = "Win*"; Include = "WinRM" }
 Invoke-Command -Session $s -ScriptBlock { Get-Service @Using:Splat }
 ```
 
-## Using local variables in PowerShell 2.0
+### Other situations where the 'Using' scope modifier is needed
+
+For any script or command that executes out of session, you need the `Using`
+scope modifier to embed variable values from the calling session scope, so that
+out of session code can access them. The `Using` scope modifier is supported in
+the following contexts:
+
+- Remotely executed commands, started with `Invoke-Command` using the
+  **ComputerName** or **Session** parameter (remote session)
+- Background jobs, started with `Start-Job` (out-of-process session)
+- Thread jobs, started via `Start-ThreadJob` (separate thread session)
+
+Depending on the context, embedded variables are copied values of the variables
+in the caller's scope. Out-of-process sessions, such as remote sessions,
+background jobs, and workflows, receive values of variables through the
+PowerShell serialization system.
+
+## Serialization of variable values
+
+Remotely executed commands and background jobs run out-of-process. PowerShell
+uses XML-based serialization and deserialization to make the values of
+variables available across the process boundaries. The serialization process
+converts objects to a **PSObject** type. The **PSObject** is general type that
```

Comment:
I suggest avoiding mentioning of PSObject altogether, because it should be considered an _invisible_ helper type.
The type that matters from a user's perspective is `System.Management.Automation.PSCustomObject`. While, regrettably, deserialize objects truly are `System.Management.Automation.PSObject` instances, "luckily" `[pscustomobject]` refers to  `System.Management.Automation.PSObject` as well (as does `[psobject]`), so I suggest using `[pscustomobject]`.

Also, this is an oversimplification: there is a fixed number of well-known types that _do_ deserialize as their original type; to quote from https://stackoverflow.com/a/59180367/45375: "
"
**This deserialization maintains *type fidelity* only for a *limited set of known types***, as specified in [MS-PSRP, the PowerShell Remoting Protocol Specification](https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-psrp/b2baf403-78aa-4f41-b140-cc4f5090cd68). That is, only instances of a fixed set of types are deserialized _as their original type_.

**Instances of all other types are _emulated_**: list-like types become `[System.Collections.ArrayList]` instances, dictionary types become `[hasthable]` instances, and other types **become method-less (properties-only) _custom objects_ (`[pscustomobject]` instances)**, whose `.pstypenames` property contains the original type name prefixed with `Deserialized.` (e.g., `Deserialized.System.Data.DataTable`), as well as the equally prefixed names of the type's _base_ types (inheritance hierarchy).
"

Reply:
What types maintain fidelity? I cannot find it listed in MS-PSRP.

Reply:
I think mentioning PSObjects is fine here.  But we should mention that some objects are rehydrated back to their original types (but of course not the original instance).  We can mention a few types as examples, but don't need to list them all.  But here is the list (note that rehydrated cert types do not include private key):

https://github.com/PowerShell/PowerShell/blob/master/src/System.Management.Automation/engine/serialization.cs#L6609